### PR TITLE
improve help text (bnc #924502), warn only once (bnc #926621)

### PR DESCRIPTION
**Pull request by gabi2**

### CODE REVIEW

File: package/yast2-iscsi-client.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jun 18 14:22:05 CEST 2015 - gs@suse.de
+
+- warn only once about non-standard initiator name (bnc #926621)
+- improve help text concerning re-discovery of targets and 
+  start-up mode (bnc #924502)
```
Comment (mvidner) [inline package/yast2-iscsi-client.changes]: I think the space after bnc will irritate the IBS checker (both bugs)

File: src/include/iscsi-client/widgets.rb [hunk 2]
```diff
@@ -419,18 +425,20 @@ def validateInitName(key, event)
         Builtins.regexpmatch(i_name, Builtins.sformat("^eui.%1%2$", reg1, reg2))
 
       if !correct
-        continue = Popup.ContinueCancel(
+        go_on = Popup.YesNoHeadline( _("Incorrect InitiatorName"),
```
Comment (mvidner) [inline src/include/iscsi-client/widgets.rb]: Space missing in Initiator Name, unless it is common usage in the iSCSI world.

File: package/yast2-iscsi-client.changes (hunk 1 above)
Comment (gabi2) [inline package/yast2-iscsi-client.changes]: I thought space between # and number is forbidden.
Will change it and also change bnc to bsc.

Comment (mvidner) [inline package/yast2-iscsi-client.changes]: Sure, "tag# 42" is even worse than "tag #42". I meant to use "tag#42".

File: src/include/iscsi-client/helps.rb [hunk 3]
```diff
@@ -88,9 +88,10 @@ def initialize_iscsi_client_helps(include_target)
           ),
         # table of connected targets
         "server_table"   => _(
-          "List of current sessions. To add a new target, select it and press <b>Add</b>.\n" +
-            "To remove it, press <b>Log Out</b>.\n" +
-            "To change the start-up status, press <b>Toggle</b>.\n"
+          "<p>List of current sessions.<br>
```
Comment (mvidner) [inline src/include/iscsi-client/helps.rb]: Please use 80 characters per line and mimic the original formatting. (BTW Rubocop prefers joining string literals with a backslash but that can wait until we apply Rubocop tho this package)

File: src/include/iscsi-client/helps.rb [hunk 4]
```diff
@@ -88,9 +88,10 @@ def initialize_iscsi_client_helps(include_target)
...
+Use the <b>Add</b> button to get additional targets. A discovery is started to detect new targets and the start-up mode of already connected targets keeps unchanged.<br>
+Use <b>Log Out</b> to cancel the connection and with it remove the target from list.\n
```
Comment (mvidner) [inline src/include/iscsi-client/helps.rb]: A missing \<br> here?

File: src/include/iscsi-client/helps.rb [hunk 5]
```diff
@@ -140,7 +141,15 @@ def initialize_iscsi_client_helps(include_target)
...
         # list of discovered targets
         "discovered"     => _(
-          "List of discovered targets. Start a new <b>Discovery</b> or <b>Connect</b> to any target."
+          "<p>This screen shows the list of discovered targets.<br>
+Use the <b>Discovery</b> button to get available iSCSI targets (from server specified in pop-up).<br>
+<b>Log In</b> to a target to establih the connection. If login was successful the column <i>Connected</i> shows status 'True' and the target will appear on the <i>Connected Targets</i> screen.<br>
+<b>Please note:</b> Starting the <b>Discovery</b> again means doing a re-discovery of targets which possibly will change the start-up mode of already connected targets (to default 'manual').
+Switch to <i>Connected Targets</i> screen and use the <b>Add</b> button if you want to add new targets without changing the start-up mode.<br>
+To remove a target use the <b>Delete</b> button. <b>Hint:</b>
+Removing of targets is only possible for not connected onces. If required <b>Log Out</b> at <i>Connected Targets</i>
```
Comment (mvidner) [inline src/include/iscsi-client/helps.rb]: Add comma after "If required"

File: src/include/iscsi-client/helps.rb [hunk 6]
```diff
@@ -140,7 +141,15 @@ def initialize_iscsi_client_helps(include_target)
           ),
         # list of discovered targets
         "discovered"     => _(
-          "List of discovered targets. Start a new <b>Discovery</b> or <b>Connect</b> to any target."
+          "<p>This screen shows the list of discovered targets.<br>
+Use the <b>Discovery</b> button to get available iSCSI targets (from server specified in pop-up).<br>
```
Comment (mvidner) [inline src/include/iscsi-client/helps.rb]: I wanted to say "from *a* server specified in *a* pop-up" but then I checked and found that it is a full size dialog, not a pop-up. So I'd rephrase to "Use the \<b>Discovery\</b> button to enter a server address and get available iSCSI targets for it.\<br>"

File: src/include/iscsi-client/helps.rb [hunk 7]
```diff
@@ -140,7 +143,20 @@ def initialize_iscsi_client_helps(include_target)
           ),
         # list of discovered targets
         "discovered"     => _(
-          "List of discovered targets. Start a new <b>Discovery</b> or <b>Connect</b> to any target."
+          "<p>This screen shows the list of discovered targets.</p>" \
+          "<p>Use the <b>Discovery</b> button to get available iSCSI targets (from server " \
+          "specified in pop-up).<br>" \
```
Comment (mvidner) [inline src/include/iscsi-client/helps.rb]: forgot to fix this one, or kept it intentionally?

Comment (mvidner) [inline src/include/iscsi-client/helps.rb]: I mean, my previous comment was:
> I wanted to say "from **a** server specified in **a** pop-up" but then I checked and found that it is a full size dialog, not a pop-up. So I'd rephrase to "Use the <b>Discovery</b> button to enter a server address and get available iSCSI targets for it.\<br>"

Comment (gabi2) [inline src/include/iscsi-client/helps.rb]: Thanks for spotting this - mixed it up with the pop-up for initiator name.

File: package/yast2-iscsi-client.changes [hunk 8]
```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Thu Jun 18 14:22:05 CEST 2015 - gs@suse.de
+
+- warn only once about non-standard initiator name (bsc#926621)
+- improve help text concerning re-discovery of targets and 
+  start-up mode (bsc#924502, bsc#924691, bsc#924693) and about
+  authentication settings (bsc#924685)
```
Comment (mvidner) [inline package/yast2-iscsi-client.changes]: It looks like an improved help text can fix many bugs :)

**Comment by mvidner** [comment]
LGTM, thank you!
